src/Microsoft.Graph.Cli.Core/IO/AuthenticationCacheUtility.cs

```diff
@@ -135,4 +238,25 @@ public AuthenticationIdentifierException(string? message, Exception? innerExcept
         {
         }
     }
+
+    class IdentityLogger : IIdentityLogger
```

Comment:
The class is unused/unreferenced. Should we remove it?

Reply:
Yeah. I used it to troubleshoot.